ipa-core/src/hpke/mod.rs

```diff
@@ -126,6 +126,38 @@ pub fn open_in_place<'a, R: PrivateKeyRegistry>(
     Ok(pt)
 }
 
+/// Version of `open_in_place` that doesn't require Info struct.
+/// ## Errors
+/// If ciphertext cannot be opened for any reason.
+pub fn hybrid_open_in_place<'a, R: PrivateKeyRegistry>(
```

Comment:
Is it worth avoid duplication between this and `open_in_place` ?

Reply:
Probably. I didn't want to step on anyone's toes but it would make sense to have just one function with the new interface that doesn't require an Info struct

Reply:
I'll just go ahead and do it, looks like these functions are only used one other place and it's easy enough to change